students/14th/team_1/seungje/project_westagram/account/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from .views import SignUp,SignIn 
+urlpatterns=[
+path('signup', SignUp.as_view()),
+path('signin', SignIn.as_view())
+]
```

Comment:
```suggestion
    path('signup', SignUp.as_view()),
    path('signin', SignIn.as_view())
]
```
리스트 내부에 존재하는 요소들이기 때문에 들여쓰기로 구분해주시는 것을 추천드립니다.